pkg/asset/machines/aws/machinedeployments.go

```diff
@@ -13,8 +16,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// MachineSets returns a list of machinesets for a machinepool.
-func MachineSets(config *types.InstallConfig, pool *types.MachinePool, role, userDataSecret string) ([]clusterapi.MachineSet, error) {
+// MachineDeployments returns a list of machinedeployments for a machinepool.
+func MachineDeployments(config *types.InstallConfig, pool *types.MachinePool, role, userDataSecret string) ([]clusterapi.MachineDeployment, error) {
```

Comment:
Hive shares this function to maintain post-install, the changes here look fine from my POV.